static/openapi/p3-spec.json

```diff
@@ -0,0 +1,3479 @@
+                  "type": "object",
+                  "required": [
+                    "testCount",
+                    "testsPassed",
+                    "testsFailed"
+                  ],
+                  "properties": {
+                    "testCount": {
+                      "type": "number",
+                      "example": 1
+                    },
+                    "testsPassed": {
+                      "type": "number",
+                      "example": 0
+                    },
+                    "testsFailed": {
+                      "type": "number",
+                      "example": 1
+                    },
+                    "error": {
+                      "type": "string",
+                      "example": "N/A",
+                      "description": "Error message if the test suite failed, otherwise not required."
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "responses": {
+            "200": {
+              "description": "Successfully saved test suite result.",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "type": "object",
+                    "properties": {
+                      "message": {
+                        "type": "string",
+                        "example": "Test suite result saved successfully."
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "400": {
+              "description": "Bad request, invalid input parameters."
+            },
+            "401": {
+              "description": "Authorization failure due to invalid authentication credentials."
+            },
+            "403": {
+              "description": "Either the API key doesn't have permissions or it is deactivated."
+            },
+            "404": {
+              "description": "The validation with the given ID was not found."
+            },
+            "500": {
+              "description": "Internal server error."
+            }
+          }
+        }
+      },
+      "/api/p3/v1/validations/testSuite/policy/{policyId}": {
+        "post": {
+          "summary": "Process Manual Tests by Policy ID",
+          "tags": [
+            "Validations"
+          ],
+          "security": [
+            {
+              "ApiKeyAuth": []
+            }
+          ],
+          "description": "Processes manual tests for a given policy ID and returns test suite results.",
+          "operationId": "testSuiteByPolicyId",
+          "parameters": [
+            {
+              "name": "policyId",
+              "in": "path",
+              "required": true,
+              "schema": {
+                "type": "string"
+              },
+              "description": "The unique identifier of the policy for which to process manual tests. If omitted, tests for all policies may be processed."
```

Comment:
"may"?  Why would it not?  It either runs against one policy or all policies.  Confusing.